### PR TITLE
0.1.0 Release

### PR DESCRIPTION
**Pull request by sgsutcliffe**
Initial release of `staramrnf`, or staramr nextflow pipeline, is a nextflow wrapper of [staramr](https://github.com/phac-nml/staramr/). Doing some last set of testing before the release into main. 

## PR checklist

- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Tested on Azure
- [ ] Tested in IRIDA Next
- [ ] Document an additonal tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
-# nf-core/staramr: Changelog
+# phac-nml/staramrnf: Changelog
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## [0.1.0] - 2024-12-08
```
Comment (apetkau) [inline CHANGELOG.md:6]: Can you change to today's date (also format like yyyy-mm-dd).

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks great. Just one small change prior to release 😄

File: CHANGELOG.md (hunk 1 above)
Comment (sgsutcliffe) [reply CHANGELOG.md]: It feels like the year is going by fast but thankfully not that fast! Made the change 4f2d48b

**Review by apetkau — APPROVED** [review]
Looks great to me.

**Pull request merged**